packages/backend/src/services/SearchService/SearchService.ts

```diff
@@ -82,6 +82,14 @@ export class SearchService {
             dashboards: results.dashboards.filter(filterItem),
             savedCharts: results.savedCharts.filter(filterItem),
             spaces: results.spaces.filter(filterItem),
+            pages: user.ability.can(
+                'view',
+                subject('Analytics', {
+                    organizationUuid,
+                }),
+            )
+                ? results.pages
+                : [], // For now there is only 1 page and it is for admins only
```

Comment:
We only need to show this page to admins (and a future dashboard analytics) , if we add another page for other users we'll have to think how to handle permissions.